packages/storybook-vue/src/stories/RadioButton/RadioButtonWithFieldsetAndLegend.stories.mdx

```diff
@@ -0,0 +1,66 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { RadioButton } from "@utrecht/component-library-vue";
+import { argTypes, radioButtonWithFieldsetAndLegendData } from "./argTypes.ts";
+
+<Meta
+  id="vue-radio-button-with-field-set-and-legend"
+  title="Vue.js Component/RadioButton/ RadioButton With Fieldset And Legend"
+  component={RadioButton}
+  argTypes={argTypes}
+/>
+
+# RadioButton With Fieldset And Legend
+
+export const Template = (args) => ({
+  components: { RadioButton },
+  setup() {
+    return { args, options: args.radioButtons };
+  },
+  template: `
+  <form>
+  <FormFieldset>
+  <FormFieldsetLegend>Kies locatie</FormFieldsetLegend>
+  <FormField v-for="{id, label} in options" :key="id">
+    <FormLabel :for="id" type="radio" :checked="args.checked" :disabled="args.disabled">
+        {{label}}
+        <RadioButton v-bind="args" :id="id" @change="action" />
+    </FormLabel>
+     </FormField>
+    </FormFieldset>
+  </form>
+  `,
```

Comment:
`name` attribute is also important to have on the `<input>` to make sure the radio buttons are part of the same group.